source/elements/oneTBB/source/containers/concurrent_unordered_set_cls/deduction_guides.rst

```diff
@@ -66,31 +53,50 @@ class template argument deduction (since C++17):
 
     template <typename T,
               typename Allocator>
-    concurrent_unordered_set( std::initializer_list<value_type>,
-                              map_size_type, Allocator )
+    concurrent_unordered_set( std::initializer_list<T>,
+                              set_size_type,
+                              Allocator )
     -> concurrent_unordered_set<T,
-                                std::hash<Key>,
-                                std::equal_to<Key>,
+                                std::hash<T>,
+                                std::equal_to<T>,
+                                Allocator>;
+
+    template <typename T,
+              typename Allocator>
+    concurrent_unordered_set( std::initializer_list<T>,
+                              Allocator )
+    -> concurrent_unordered_set<T,
+                                std::hash<T>,
+                                std::equal_to<T>,
                                 Allocator>;
 
     template <typename T,
               typename Hash,
               typename Allocator>
-    concurrent_unordered_set( std::initializer_list<value_type>,
-                              map_size_type, Hash, Allocator )
+    concurrent_unordered_set( std::initializer_list<T>,
+                              set_size_type,
+                              Hash,
+                              Allocator )
     -> concurrent_unordered_set<T,
                                 Hash,
-                                std::equal_to<Key>,
+                                std::equal_to<T>,
                                 Allocator>;
 
-Where the type ``map_size_type`` refers to the ``size_type`` member type of the deduced ``concurrent_unordered_set``
+Where the type ``set_size_type`` refers to the ``size_type`` member type of the deduced ``concurrent_unordered_set``
 and the type alias ``iterator_value_t`` is defined as follows:
```

Comment:
```suggestion
and the alias type ``iterator_value_t`` is defined as follows:
```

Reply:
I think it would be better to use `type alias`, because this term is described in the standard: https://en.cppreference.com/w/cpp/language/type_alias

Reply:
Sure, then let's keep it